tephra-core/src/main/java/org/apache/tephra/TransactionManager.java

```diff
@@ -763,15 +781,22 @@ private long getNextWritePointer() {
    * transaction moves it to the invalid list because we assume that its writes cannot be rolled back.
    */
   public Transaction startLong() {
+    return startLong(null);
+  }
+
+  /**
+   * Starts a long transaction with a client id.
+   */
+  public Transaction startLong(String clientId) {
```

Comment:
Eventually we want all client's to pass in a client i, hence the deprecation. If we allow nulls then that defeats the purpose, no?